lib/project_types/php/commands/connect.rb

```diff
@@ -1,16 +1,21 @@
 # frozen_string_literal: true
-
 module PHP
-  module Commands
+  class Command
     class Connect < ShopifyCli::SubCommand
+      prerequisite_task ensure_project_type: :php
```

Comment:
I think this needs to be removed - if the user is attempting to connect a PHP project to the CLI, it likely means there's no `.shopify-cli.yml` file in the directory, which is what this `ensure_project_type` task does ... it checks to see if this command is being run in a project that has `project_type` set to `php` in the `.shopify-cli.yml` file ... which likely won't exist if the user is trying to run `shopify php connect`.

I think this is a bug that needs to be fixed across all the `connect` commands.

Reply:
That makes a lot of sense to me, and I found it quite weird that this requirement was there in the first place. Just removing this line makes it work for non-project folders!